l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -262,42 +318,27 @@ def get_ob_totals(self):
             "and l.account_id = a.id "
             "and l.parent_state = 'posted' "
             "and l.date < %s "
-            "and l.company_id=%s "
+            "and l.company_id = %s "
             "and t.include_initial_balance = true "
-            "and t.id!=%s",
+            "and t.id != %s",
             (self.date_start, self.company_id.id, cye_type_id),
         )
-        row = self.env.cr.fetchall()[0]
+        row = self.env.cr.fetchone()
+        credit, debit, account_count = (
+            (row[0] or 0.0, row[1] or 0.0, row[2] or 0) if row else (0.0, 0.0, 0)
```

Comment:
Naive question, but how do we know that `row` contains at least 3 elements?

Reply:
Because there are three columns, and we have an SQL statement that always returns exactly one row.